docs/ios/bazel.md

```diff
@@ -0,0 +1,118 @@
+
+Copy `productscience.yaml` to your project's root directory. The file should sit next to your Bazel `WORKSPACE` file.
+
+**Example**
+
+Assume that your project's root directory is `/code/MyApp`. After adding `productscience.yaml`, your directory structure should resemble the following:
+
+```shell
+/code
+└── MyApp
+    └── productscience.yaml
+    └── WORKSPACE
+    └── ...
+```
+
+## 2. Install `PSCliCodeInjector`
+
+Download the latest installer package (named `PSCliCodeInjector.pkg`) from our [public plugin repo](https://github.com/product-science/PSios/releases).
+
+Double-click the downloaded `.pkg` to install the `PSCliCodeInjector` command line tool. By default, the tool will be installed to `/usr/local/bin`.
+
+## 3. Install `buildozer`
+
+`buildozer` is a build tool maintained by the Bazel team. `PSCliCodeInjector` uses this tool to configure your
+project's `BUILD` files during injection.
+
+Before running `PSCliCodeInjector` for the first time, you'll need to install `buildozer` on the host computer.
+Instructions for installation can be found on the project's [repository](https://github.com/bazelbuild/buildtools/tree/master/buildozer).
+
+## 4. Run `PSCliCodeInjector`
+
+`PSCliCodeInjector` adds Product Science's instrumentation to your project's source code.
+
+Before any changes are made to your project, `PSCliCodeInjector` will create a copy of your project's directory and save it to a backup location. `PSCliCodeInjector` will then add Product Science's instrumentation to you original project's source code. The backup directory will contain the original, un-instrumented code.
+
+:bulb: **IMPORTANT**: When you want to create an instrumented build, be sure to use the original project directory and not the backup directory.
+
+### A. Basic use
+
+**Format**
+
+```shell
+PSCliCodeInjector "<root-folder>" \
+  --console-build-command "<console-build-command>"
+```
+
+There are only two required parameters when running `PSCliCodeInjector`:
+
+1. `root-folder`: This is the path to your project's root directory. There must be a Bazel `WORKSPACE` file at the top level of this directory.
+2. `console-build-command`: This is the build command that the tool will use to confirm that your project compiles successfully after injection. This command will be run from your project's root directory (the directory that contains your `WORKSPACE` file).
+
+**Example**
+
+Continuing with the previous example - As before, your project's root directory is `/code/MyApp`. 
+
+Assume that you use the command `bazel build //ios-app:ios-app` to build your project.
+ 
+Given the information above, your default `PSCliCodeInjector` command will be:
+
+```shell
+PSCliCodeInjector "/code/MyApp" \
+  --console-build-command="bazel build //ios-app:ios-app"
+```
```

Comment:
Do we have this MyApp in demoapps? To attach project on which client can check it?